apps/prairielearn/elements/pl-python-variable/pl-python-variable.py

```diff
@@ -66,7 +68,10 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     indent = pl.get_integer_attrib(element, "indent", INDENT_DEFAULT)
     depth = pl.get_integer_attrib(element, "depth", DEPTH_DEFAULT)
     width = pl.get_integer_attrib(element, "width", WIDTH_DEFAULT)
-    compact = pl.get_boolean_attrib(element, "compact", COMPACT_DEFAULT)
+
+    compact_default = pl.get_boolean_attrib(element, "compact", COMPACT_DEFAULT)
```

Comment:
This can cause an error if `compact` actually exists, so you may want to keep it in a try-catch block or capture it some other way (string?). Also include a comment?

Reply:
I don't think that's quite right? If `compact` is present, it will just read it and use it as the default value for `compact-sequences` when that's read in. We kept `compact` in the check for attributes.

Reply:
Added comment here: https://github.com/PrairieLearn/PrairieLearn/pull/8468/commits/9ecc71d3f5fe07eb7dcac365c51a937421e28a28

Reply:
If compact gets an lxml round trip its value will be changed to a blank string, which is invalid as a boolean.

Reply:
Ahh that's a good point. I guess the try block should just print an error message then.

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/8468/commits/0ff5c4ec50b530701101034e7dfb59badb9904d3